active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,572 @@
+
+XAML
+```XAML
+<Button Content="Save" Click="OnFirstSaveButtonClick"></Button>
+<muxc:TeachingTip> 
+    <muxc:TeachingTip x:Name="AutoSaveTip"
+        Title="Saving automatically"
+        Subtitle="We save your changes as you go - so you never have to."
+        Margin="80">
+    </muxc:TeachingTip>
+</muxc:TeachingTip.Attach>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+    TeachingTipExample.IsOpen=true;
+}
+```
+
+<a href="url"><img src="TeachingTipMarginSample.jpg" height="600"></a>
+
+### Point to a UI element
+
+You can change the teaching tip to point to a specific UI element by setting the TeachingTipPlacementMode property. For a targeted teaching tip, this placement will be relative to the UI element target. For a non-targeted teaching tip, this placement will relative to app window.  
+
+XAML
+```XAML
+<Button Content="Tip Example" Click="OnFirstSaveButtonClick">
+    <muxc:TeachingTip.Attach> 
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to."
+            Margin="15">
+        </muxc:TeachingTip>
+    </muxc:TeachingTip.Attach>
+</Button>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+    TeachingTipExample.IsOpen=true;
+}
+```
+
+<a href="url"><img src="TeachingTipPointerSample.jpg" height="600"></a>
+
+### Add content
+
+Content can be added to a teaching tip using the Content property. If there is more content to show than what the size of a teaching tip will allow, a scrollbar will be automatically enabled to allow a user to scroll the content area. 
+
+XAML
+```XAML
+<Button Content="Tip Example" Click="OnFirstSaveButtonClick">
+    <muxc:TeachingTip.Attach> 
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to."
+            Margin="15">
+                <CheckBox Content="Don't show tips at start up" Checked="HandleChecked" Unchecked="HandleUnchecked" />
+                <p>You can change your tip preferences in <a href="app:/item/SettingsPage">Settings</a> if you change your mind.</p>
+        </muxc:TeachingTip>
+    </muxc:TeachingTip.Attach>
+</Button>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+    TeachingTipExample.IsOpen=true;
+}
+```
+
+<a href="url"><img src="TeachingTipContentSample.jpg" height="600"></a>
+
+### Add buttons
+
+A close button is automatically added to the top of a teaching tip. However, a close button with custom text that is placed at the button of a teaching tip can be used instead by setting the TeachingTipCloseButtonKind property. A custom action button can also be added by setting ActionButtonText, ActionButtonCommand and the ActionButtonCommandParameter properties. 
```

Comment:
Add a sentence here explaining how CloseButtonKind.Auto works?

Reply:
```suggestion
A standard "X" close button is automatically added to the top of a teaching tip. However, a custom close button can be added to the bottom of a teaching tip by setting the TeachingTipCloseButtonKind property.  When this property is set to Auto, the "X" close button at the top of the tip will be preferred unless CloseButtonText is set. A custom action button can also be added by setting ActionButtonText, ActionButtonCommand and the ActionButtonCommandParameter properties. 
```

Reply:
@MikeHillberg Does this feel better?

Reply:
If I set the Close button to bottom, but don't set CloseButtonText, do I get some default text?

Reply:
Can the button be removed entirely?